Tests_v3/Dummy data/MutedUser.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+@testable import StreamChatClient
+
+extension MutedUser {
```

Comment:
`MutedUser` should be probably renamed to `MutedUserPayload`

Reply:
fixed in 1a1e81ac0ce4ffde69a610f60713cd5291b62159